### ISSUE TITLE
hCaptcha shows Popup to leave the page

### ISSUE THREAD

**Issue by veronika-g** [verbb/formie#1215]
### Describe the bug

Hi!

I added hCaptcha to my forms which works fine, but I keep getting a popup asking if I want to leave the page even though I haven't interacted with the form.

Thanks for help!
Veronika

### Steps to reproduce

1. Add form to the page
2. Use hCaptcha as integration (invisible)

### Form settings

- Multi-page form: Yes (I use both)
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.3.4

### Plugin version

2.0.18

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by veronika-g** [comment]
Hey thanks for your quick fix! It's working :)

**Comment by engram-design** [comment]
Fixed in [2.0.20](https://github.com/verbb/formie/releases/tag/2.0.20)
